README.md

```diff
@@ -227,30 +238,45 @@ See the format of the config file below with sample values:
   "INFERENCE_ACTIVATION": true
 }
 ```
-Below is the description of each setting:
-
-1. `BUILD_COMMAND`: The command to execute `NullAway` for the project. (This command must include changing directory to target project from root)
-2. `INITIALIZER`: Fully qualified name of the `Initializer` annotation to inject on detected initializer methods.
-3. `NULLABLE`: Fully qualified name of the `Nullable` annotation.
-4. `OUTPUT_DIR`: Directory where the serialized output of NullAway should be written.
-5. `CONFIG_PATHS`: Array of JSON objects where each contains the path to nullaway and scanner Error Prone checker. The first element in this array will be considered as the target module and rest as downstream dependencies.
-6. `PARALLEL_PROCESSING`: Enables parallel processing of fixes within an iteration.
-7. `CACHE_IMPACT_ACTIVATION`: Enables fixes impacts caching for next cycles.
-8. `CACHE`: if set to `true`, cache usage will be enabled.
-9. `BAILOUT`: if set to `true`, Annotator will bailout from the search tree as soon as its effectiveness hits zero or less, otherwise it will completely travers the tree until no new fix is suggested
-10. `DEPTH`: The depth of the analysis.
-11. `REDIRECT_BUILD_OUTPUT_TO_STDERR`: If true, build output will be redirected to `STD Error`.
-12. `EXHAUSTIVE_SEARCH`: If true, the exhaustive search will be activated.
-13. `OUTER_LOOP`: If true, the outer loop will be enabled.
-14. `DOWNSTREAM_DEPENDENCY:ACTIVATION`: Controls downstream dependency feature activation.
-15. `DOWNSTREAM_DEPENDENCY:BUILD_COMMAND`: Single command to build all downstream dependencies.
-16. `DOWNSTREAM_DEPENDENCY:LIBRARY_MODEL_LOADER_PATH`: Path to NullAway library models loader.
-17. `DOWNSTREAM_DEPENDENCY:ANALYSIS_MODE`: Analysis mode for downstream dependencies. Can be [default|upper_bound|lower_bound|strict]
-18. `FORCE_RESOLVE`: If true, remaining unresolved errors will be silenced using suppression annotations.
-19. `INFERENCE_ACTIVATION`: If true, inference of `@Nullable` annotation will be activated.
-20. `PROCESSORS:LOMBOK:ACTIVATION`: If true, potentially impacted regions will be extended with generated code by Lombok.
-
-Pass the path to the config file above with `-p,--path` argument to `core.jar` and no other flag is required.
-
-
-To see all flags description simply run the jar with `--help`.
+Here is a template command you can use to run Annotator from the CLI, using config.json - 
+```bash
+java -jar /path/to/annotator-core.jar -p path_to/config.json
+```
+Provide the path to the configuration file above with the `-p,--path` argument to `core.jar`, and no other flags are needed.
+
+To view descriptions of all flags, simply run the JAR with the `--help` option.
+
+## NullAway Compatibility
+
+- `Annotator` version `1.3.6` is compatible with `NullAway` version `0.10.10` and above.
+
+
+## Features 🧱
```

Comment:
Maybe we can delete emojis for now :)

Reply:
Updated it, removed all the emojis (Resolving all comments, please resolve this one after review)